teams/tidb/membership.json

```diff
@@ -4,6 +4,7 @@
     "maintainers": [
         "3pointer",
         "XuHuaiyu",
+        "YuJuncen",
```

Comment:
Case sensitive sorting?

Reply:
Yes, following the current collation: https://github.com/pingcap/community/blob/master/teams/tidb/membership.json